linker-diff/src/lib.rs

```diff
@@ -158,6 +158,20 @@ impl Config {
             .into_iter()
             .map(ToOwned::to_owned),
         );
+
+        #[cfg(target_arch = "aarch64")]
```

Comment:
It'd be good to decide this at runtime rather than at compile time. e.g. https://github.com/davidlattimore/wild/pull/268. But I'm happy to leave this as is for now in order to get this merged.

Reply:
Sure, that's definitely something that needs to be covered in #267.